### DISCUSSION TITLE
todo

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#133] [Announcements]
# todo

主题的开发更新计划

https://theme.sugarat.top/todo.html

**Comment by Lazydd** [comment]
老哥，你这个评论功能是怎么实现的

**Reply by ATQQ** [reply]
这个在文档里有提到http://theme.sugarat.top/config/global.html#comment

用的 https://giscus.app/zh-CN

**Comment by kakatelo** [comment]
1111

**Reply by ATQQ** [reply]
222

**Comment by pejaz** [comment]
点击标签滚动回去顶部的时候可以加点滚动动画。

**Reply by pejaz** [reply]
公告关闭后可以配置不闪烁吗。或者直接隐藏，一直闪烁比较影响阅读

**Reply by ATQQ** [reply]
> 公告关闭后可以配置不闪烁吗。或者直接隐藏，一直闪烁比较影响阅读

可以关掉呀，配置里有说明：https://theme.sugarat.top/config/global.html#popover

```js
const blogTheme = getThemeConfig({
  popover: {
    twinkle: false
  }
})
```

**Reply by ATQQ** [reply]
滚动这个我记一下看看。

**Comment by xiyuvi** [comment]
很漂亮的主题，大佬考虑在文章页增加 上一篇 下一篇 功能吗

**Reply by ATQQ** [reply]
都能做，只是时间有限推进得比较慢，可以来贡献代码呀

**Comment by HadryChen** [comment]
老哥，请问下导航栏如何实现类似于`标签`点击页面的展示啊，或者导航菜单能否跳转到标签的文章列表页？

**Reply by ATQQ** [reply]
导航配对应标签的路由就行加上query参数
/xxx?a=b

**Reply by HadryChen** [reply]
例如我现在有个标签是：`AI`，我点击标签按钮时，跳转的是：`http://localhost:5173/?tag=AI&type=`，但是我在导航栏菜单这么配置并不生效，`{ text: "AI", link: "/?tag=AI&type=" }` 或者 `{ text: "AI", link: "/?tag=AI" }`都没生效，烦请赐教

**Reply by ATQQ** [reply]
那我下来看看吖。

**Reply by HadryChen** [reply]
好像是个bug，进入首页时，`首页`菜单与 `AI`菜单的样式都是`active`，您可以看下

**Reply by ATQQ** [reply]
看了一下 应该VitePress 默认行为是这样，顶导里的相同路由时 不会再触发路由跳转

**Reply by HadryChen** [reply]
我看只要是导航栏菜单中加了这个跳转路径`/?tag=xxx`的，样式都会被默认激活成`active`，这是我所发现的，不知道老哥能否搞定这个问题 O(∩_∩)O哈哈~

**Reply by ATQQ** [reply]
加个中间页重定向
```md
---
publish: false
---

<script setup lang="ts">
import { useRouter } from 'vitepress'
const router = useRouter()

if (typeof window !== 'undefined') {
  const url = new URL(window.location.href)
  const tag = url.searchParams.get('tag')
  if (tag) {
    router.go(
      `${window.location.origin}/?tag=${tag}`
    )
  }
}
</script>

```

<img width="974" height="506" alt="image" src="https://github.com/user-attachments/assets/72a66cc4-028b-4e4e-b161-5fbc5c5b4068" />

配置页
```json
          { text: 'AI', link: '/redirect-tag?tag=AI' }
```

**Reply by HadryChen** [reply]
谢谢博主 赞
